### DISCUSSION TITLE
偶然触发的闪退原因，供各位参考

### DISCUSSION THREAD

**Discussion by varLiu** [xiaye13579/BBLL#1038] [Show and tell]
品牌：海信
CPU型号：MT7642
CPU架构：armeabi-v7a
系统版本：Android 9
近日闲来无事，想在TV上倒腾科学上网，安装clash后提示缺少谷歌框架，然后安装谷歌三件套，配置好clash，但是一直出现dns连接失败的问题，解决不了无奈放弃。
这些都不是重点，然后想打开bbll看看番，竟然闪退！以前bbll在电视上从未出现闪退，然后下载多个旧版都一样闪退，最终直觉告诉我跟谷歌框架有关，然后卸载三件套，问题解决。
但是，我的手机是也是有谷歌框架的，安装bbll就不存在闪退现象，也许这和CPU架构以及系统版本有关，我的手机CPU架构是arm64-v8a，系统是13。

**Comment by binty88git** [comment]
折腾了半天，才看到你这评论。 我的网络盒子zidoo z9s也是存在google套件，安装BBLL后打开就显示“已停止运行”，无法使用。望修复

**Comment by XiaoluoFoxington** [comment]
> 品牌：海信
> CPU型号：MT7642
> CPU架构：armeabi-v7a
> 系统版本：Android 9
> 近日闲来无事，想在TV上倒腾科学上网，安装clash后提示缺少谷歌框架，然后安装谷歌三件套，配置好clash，但是一直出现dns连接失败的问题，解决不了无奈放弃。
> 这些都不是重点，然后想打开bbll看看番，竟然闪退！以前bbll在电视上从未出现闪退，然后下载多个旧版都一样闪退，最终直觉告诉我跟谷歌框架有关，然后卸载三件套，问题解决。
> 但是，我的手机是也是有谷歌框架的，安装bbll就不存在闪退现象，也许这和CPU架构以及系统版本有关，我的手机CPU架构是arm64-v8a，系统是13。

BBLL在我的电视上也存在莫名偶发的闪退，是看着看着突然回到主屏幕，无任何征兆。不过我没有安装谷歌框架。
